### DISCUSSION TITLE
Change mx-*, px-*, border-x-*, my-*, py-* and border-y-* utilities

### DISCUSSION THREAD

**Discussion by PositiveUser** [tailwindlabs/tailwindcss#7069] [Ideas]
@adamwathan. @simonswiss.

---

`mx-0`

```css
margin-inline: 0;
```
Or
```css
margin: revert 0;
```

---

`px-0`

```css
padding-inline: 0;
```
Or
```css
padding: revert 0;
```

---

`border-x-0`

```css
border-inline-width: 0;
```

`border-x-gray-200`

```css
border-inline-color: rgb(229 231 235 / var(--tw-border-opacity));
```

---

`my-0`

```css
margin-block: 0;
````
Or
```css
margin: 0 revert;
```

---

`py-0`

```css
padding-block: 0;
```
Or
```css
padding: 0 revert;
```

---

`border-y-0`

```css
border-block-width: 0;
```

`border-y-gray-200`

```css
border-block-color: rgb(229 231 235 / var(--tw-border-opacity));
```

---

Same for other `mx-*`, `px-*`, `my-*`, `py-*`, `border-x-*` and `border-y-*` utilities.

This will decrease the file size of generated CSS

**Comment by windyclory** [comment]
That file size doesn't matter and those properties aren't beginner-friendly.
